game/dchar/erin.rpy

```diff
@@ -19,7 +19,7 @@ init -4 python:
         ("Nice to see you around!", "happy")]
         
     linesTalking[iC][MS_GOOD_FRIEND] += [
-        ("A nice day to you.", "smile")]
+        ("Hey! Nice day today, isn't it?", "smile")]
```

Comment:
"smile" is not a valid emotion in this game, please change to something from emotions.txt or just remove ``,"smile``